### DISCUSSION TITLE
Is there a way to start processing a file but then abort/pause it somehow?

### DISCUSSION THREAD

**Discussion by thibaultmol** [ggml-org/whisper.cpp#913] [Q&A]
Like, if you're running whisper.cpp on an hour long audio file. And you just want to pause it for a bit (cause you're on a laptop and want to save some battery life till you're back on AC power for ex).
Is there a system so that you can easily 'continue' the transription at a later point where it left off?

**Comment by indokid9999** [comment]
In win 8  ( and maybe 10 and 11) you can run "resmon.exe". Find the the name of the process you want to pause, in your case "main.exe" and right-click should show the  "suspend process" option.

On linux , get the <pid> (process_id)  of the application you want to suspend using

$ ps aux | grep “main”
 
and then use  "kill -STOP <PID>" to pause the process and "kill -CONT <PID>" to resume it.

hope it helps.

**Comment by ArthurPeabody** [comment]
I use linux, kill jobs that go on too long, restart with the -ot switch.  Don't know nuthin' 'bout 'Windoze'.
